GothicVRProject/GothicVR
# Optimization/move to zenkit

It is done!

# To test
1. Start game with Production Feature flags and spawn Bloodwyn + Logs (see picture)
![image](https://github.com/GothicVRProject/GothicVR/assets/120568393/8c9517b9-c608-41bc-9917-ca078090ca66)
2. Start game and play 
3. Look for
    4. Crashes - If not, we're good
    5. Some other weird stuff or errors (Not expected)

# Checklist

## Scene

# Testing

## Dependencies
